orca-mine/src/main/groovy/com/netflix/spinnaker/orca/mine/pipeline/MonitorCanaryStage.groovy

```diff
@@ -37,22 +39,22 @@ class MonitorCanaryStage implements StageDefinitionBuilder, CancellableStage {
 
   @Autowired MineService mineService
   @Autowired CanaryStage canaryStage
+  @Autowired CanaryDisableClusterStage canaryDisableClusterStage
 
   @Override
   void taskGraph(@Nonnull StageExecution stage, @Nonnull TaskNode.Builder builder) {
     builder
       .withTask("registerCanary", RegisterCanaryTask)
       .withTask("monitorCanary", MonitorCanaryTask)
-      .withTask("disableCanaryCluster", DisableCanaryTask)
```

Comment:
These tasks were broken up into two extra stages. The disable canary stage and the destroy canary stage.